thinker.py

```diff
@@ -18,12 +19,16 @@ def __init__(self):
 		self.opp_active_mon = ''
 		self.opp_pokemon_dict = dict()
 
+#Chooses next move based off of all possible moves
 	def get_next_move(self, is_forced_switch, is_forced_stay):
 		actions = self.__get_possible_actions(is_forced_switch, is_forced_stay)
 		selection = self.__choose_next_action(actions)
 		return selection
 
+#Isolates usable moves and switchable mons
 	def __get_possible_actions(self, is_forced_switch, is_forced_stay):
+#Are there any force switch moves? I thought roar and whirlwind autopicked one. Would i just be teleport
```

Comment:
If your active poke faints you have to switch. This just stops it from trying to use a move of a pokemon that just fainted